src/tc-shortcut/tc-shortcut.hpp

```diff
@@ -0,0 +1,313 @@
+
+#ifndef TC_SHORTCUT_HPP
+#define TC_SHORTCUT_HPP
+
+
+#include "hashutil.h"
+#include "./tc-sym.hpp"
+
+template <typename HashFamily = hashing::TwoIndependentMultiplyShift>
```

Comment:
It is fine to use TwoIndependentMultiplyShift... I just wonder (and this is really just curiosity), did you try with SimpleMixSplit as well? If not, no problem - I will try myself.